src/utils/viewer/viewer.cpp

```diff
@@ -41,18 +41,17 @@
 
 #include "esp/gfx/configure.h"
 
-using namespace Magnum;
-using namespace Math::Literals;
-using namespace Corrade;
-
-using namespace esp;
-
 constexpr float moveSensitivity = 0.1f;
 constexpr float lookSensitivity = 11.25f;
 constexpr float rgbSensorHeight = 1.5f;
 
 namespace {
 
+// for ease of access
+namespace MnGL = ::Magnum::GL;
+namespace CorrUtil = ::Corrade::Utility;
+namespace MathLits = ::Magnum::Math::Literals;
```

Comment:
Um... can we not, please? :) There's a `Mn` / `Cr` shorthand that is quite well-known at this point, but these will only cause confusion to unaware readers. So instead, like in other files

```cpp
namespace Cr = Corrade;
namespace Mn = Magnum;
```

and keeping those in the global namespace shouldn't hurt either.

`Mn::GL`, `Cr::Utility` are not much longer than `MnGL` or `CorrUtil`, and you can do `using Mn::Math::Literals` directly here -- it doesn't (and will not) bring any literals potentially conflicting with the standard ones.

Reply:
Thanks for suggestions, I was unaware of the prevalence of existing aliases.  Normally i would be uncomfortable putting this kind of thing (or much of anything, really) in the global namespace, but if this is acceptable, then I'll certainly do it that way :).

Reply:
To provide more context -- on python side it's similar (`import magnum as mn`), and we refrain from putting those in headers as it confuses Doxygen (it can't then properly link the shorthand types to upstream docs). Aliases in global namespaces in cpp files are fine tho, having them always in the global namespace also makes it easier for the IDE autocompletion engine.